### DISCUSSION TITLE
Github Copilot access has ended!!

### DISCUSSION THREAD

**Discussion by MalikOwais123** [community/community#48004] [GitHub Education] [answered]
Hello folks,

I am verified student at github students developers club and I used githb copilot for my daily hobbies project as free because i have pro badge in github.
But apparently my github copilot access has been ended idk how or why, Any one there who can help me out.

**Comment by samus-aran** [comment]
Hi @MalikOwais123 👋 

Are you still experiencing this issue? What errors are you seeing on your IDE or dashboards?

**Reply by MalikOwais123** [reply]
Actually I got an error showing your server can not connected, and when i look to my mails where there has been a mail from github account that mentioned my github copilot account access has been ended and i need to purchase copilot in order to continue using copilot.

**Comment by AsafMeizner** [comment, marked as answer]
sometimes the GitHub copilot "license" ends before the student benefits, you can either reapply for the student developer pack or write an email to GitHub, both will renue your copilot

hope this helps :)

**Reply by MalikOwais123** [reply]
Yes, I was wondering and idk from where i just submit new request for reverification for my students benefit, and by the time the request ahas been approved i successfully got back my copilot advantages 😇.

Thanks for response,☺️

**Reply by AsafMeizner** [reply]
No problem :)
